docs/asyncapi.yaml

```diff
@@ -127,42 +138,57 @@ components:
         expectedPaymentDate: "21.11.2025"
         adminName: "Jessica Lrrr"
       default: {}
+      minleght: 0
+      maxlenght: 100
 
     reference:
       type: string
-      description: Unique identifier (UUID) for further operations like status query
-      example: "abc123-uuid-123456"
+      description: Unique identifier for further operations like status query
+      example: "can-2a88-be-b855-a55026844MIX"
+      minleght: 2
```

Comment:
We should specify the need for a namespace prefix here.   eg `ffc-ahwr`

Also think 2 would never be enough to guarantee a unique message reference